pkg/addons/translate.go

```diff
@@ -484,6 +484,12 @@ var Translations = map[string][]struct {
 			Template: "{{ .Config.Images.Console }}",
 		},
 	},
+	"Deployment.apps/openshift-console-branding/servfile": {
```

Comment:
Did you add this file?  I didn't see it.

Reply:
Hi kwoodson, thanks for the review, no the deployment file is not there
yet. I still have a few things to round up with this PR before I remove the
[wip].